charts/postgres-operator/values-crd.yaml

```diff
@@ -93,10 +93,9 @@ configKubernetes:
   # template for database user secrets generated by the operator
   secret_name_template: "{username}.{cluster}.credentials.{tprkind}.{tprgroup}"
   # group ID with write-access to volumes (required to run Spilo as non-root process)
-  # spilo_fsgroup: 103
```

Comment:
Regarding [this](https://github.com/zalando/postgres-operator/pull/531#discussion_r289402865) comment, quotes should be removed.